util/find-doc-nits

```diff
@@ -1029,21 +1026,43 @@ my %skips = (
     'digest' => 1,
 );
 
+my %genopts; # generic options parsed from apps/include/opt.h
+
 # Check the flags of a command and see if everything is in the manpage
 sub checkflags {
     my $cmd = shift;
     my $doc = shift;
-    my %cmdopts;
+    my @cmdopts;
     my %docopts;
     my %localskips;
 
-    # Get the list of options in the command.
-    open CFH, "$openssl list --options $cmd|"
-        or die "Can list options for $cmd, $!";
+    # Get the list of options in the command source file.
+    my $src = $cmd;
+    $src = "asn1pars" if $cmd eq "asn1parse";
+    $src = "crl2p7" if $cmd eq "crl2pkcs7";
+    my $active = 0;
+    my $expect_helpstr = "";
+    open CFH, "apps/$src.c"
+        or die "Can't open apps/$src.c to list options for $cmd, $!";
     while ( <CFH> ) {
         chop;
-        s/ .$//;
-        $cmdopts{$_} = 1;
+        if ($active) {
+            last if m/^\s*};/;
+            if ($expect_helpstr ne "") {
+                next if m/^\s*#\s*if/;
+                err("$cmd does not implement help for -$expect_helpstr") unless m/^\s*"/;
+                $expect_helpstr = "";
+            } elsif (m/^\s+\{\s*"([^"]+)"\s*,\s*OPT_[A-Z0-9_]+\s*,\s*('.'|0)\s*,(.*)$/
```

Comment:
Is it worthwhile considering \s* instead of \s+ at the beginning?  Currently this checks that there is some indentation but not that it is correct, is that better or worse than not checking?

Reply:
Also here it is not really required to check that the `{`... is at the beginning of the line, so dropped the `^\s+`.